Adjust vendoring to make VTK use the external kissfft
Thanks to Nekto89 for this patch in his recipe review.


--- a/ThirdParty/kissfft/CMakeLists.txt
+++ b/ThirdParty/kissfft/CMakeLists.txt
@@ -1,9 +1,15 @@
 set(KISSFFT_SCALAR_TYPE "double")
 
-vtk_module_third_party_internal(
-  LICENSE_FILES "vtkkissfft/COPYING"
-  VERSION       "master"
-  STANDARD_INCLUDE_DIRS)
+vtk_module_third_party(
+  INTERNAL
+    LICENSE_FILES "vtkkissfft/COPYING"
+    VERSION       "master"
+    STANDARD_INCLUDE_DIRS
+  EXTERNAL
+    PACKAGE kissfft
+    TARGETS kissfft::kissfft-double
+            kissfft-double
+    STANDARD_INCLUDE_DIRS)
 
 configure_file(
   "${CMAKE_CURRENT_SOURCE_DIR}/vtk_kissfft.h.in"
--- a/Common/Math/vtkFFT.h
+++ b/Common/Math/vtkFFT.h
@@ -36,7 +36,7 @@
 #include "vtk_kissfft.h" // For kiss_fft_scalar, kiss_fft_cpx
 // clang-format off
 #include VTK_KISSFFT_HEADER(kiss_fft.h)
-#include VTK_KISSFFT_HEADER(tools/kiss_fftr.h)
+#include VTK_KISSFFT_HEADER(kiss_fftr.h)
 // clang-format on
 
 #include <vector> // For std::vector
